_layouts/main.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html height="100%" lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>{{ site.title }}</title>
+    <link rel="stylesheet" href="assets/css/all-styles.css">
+  </head>
+  <body>
+    <div class="grid-container margin-y-5">
+      <header class="grid-row page-header">
+        <div class="grid-col-12 padding-4">
+          <img src="assets/img/sam.png" width="287" height="62" alt="SAM.gov logo" />
```

Comment:
Should this image path use site.baseurl?

Reply:
> Should this image path use site.baseurl?

Not strictly necessary as it's a relative path, but for consistency's sake with the others, may as well!

Reply:
> > Should this image path use site.baseurl?
> 
> Not strictly necessary as it's a relative path, but for consistency's sake with the others, may as well!

Updated in 831b3ce.